README.md

```diff
@@ -943,6 +943,7 @@ Table of Contents
   * [Versoly](https://versoly.com/) — SaaS focussed website builder - unlimited websites, 70+ blocks, 5 templates, custom CSS, favicon, SEO and forms. No custom domain.
   * [Qovery](https://www.qovery.com) — Qovery is the simplest way to deploy your full-stack apps on AWS, GCP and Azure. It is free web hosting for developers with Database, SSL, a global CDN, and auto deploys from Git.
   * [FlashDrive.io](https://flashdrive.io) - PaaS service similar to Heroku with a developer-centric approach and all inclusive features. Free tier for static assets, staging and developer apps.
+  * [OPHosting](https://ophosting.net/?utm_source=free-for-dev) - Free Webhosting for unlimited Projects.
```

Comment:
```For the termination of an existing contract at the request of the customer, we charge a processing fee of 25.00 EUR.```

Sorry, no.

Reply:
anyways, just saying.
this is one of those fake cPanel hosts/nulled cPanel hosts.
https://cpanel.ophosting.net/

Reply:
> anyways, just saying. this is one of those fake cPanel hosts/nulled cPanel hosts. https://cpanel.ophosting.net/

it's a legit one ;)
OPHosting is a reseller of ifastnet.com.
Stop spreading lies.

Reply:
> it's a legit one ;)

Not to have an argument, but for sake please know how stuff works.
https://verify.cpanel.net/app/verify?ip=cpanel.ophosting.net
VistaPanel or vPanel is a custom panel created by Ifastnet.com to give free hosting(which will get suspended everyday)

vP is just a business tactic to buy their premium plans(if your site is suspended it will lead to an ad page of ifastnet)

Reply:
If you want me to explain more, add me on discord FatGrizzly#5803

Reply:
Nope. OPhosting isn't a reseller of iFastNet but their sub brand, MyOwnFreeHost.net commonly known as MOFH